client/my-sites/comments/comment-list/index.jsx

```diff
@@ -27,53 +34,98 @@ export class CommentList extends Component {
 
 	componentWillReceiveProps( nextProps ) {
 		if ( ! this.props.comments.length ) {
-			this.setState( { comments: nextProps.comments } );
+			this.setState( { comments: keyBy( nextProps.comments, 'ID' ) } );
+		}
+	}
+
+	showNotice = ( commentId, newStatus, previousStatus ) => {
+		const {
+			createNotice: createCommentNotice,
+			removeNotice: removeCommentNotice,
+			translate,
+		} = this.props;
+
+		const noticeId = `comment-notice-${ commentId }`;
+
+		const options = {
+			duration: 5000,
+			id: noticeId,
+			isPersistent: true,
+		};
+
+		if ( 'delete' !== newStatus ) {
+			options.button = translate( 'Undo' );
+			options.onClick = () => this.setCommentStatus( commentId, previousStatus, { showNotice: false } );
+		}
+
+		removeCommentNotice( noticeId );
+
+		switch ( newStatus ) {
+			case 'approved':
+				return createCommentNotice( 'is-success', translate( 'Comment approved.' ), options );
+			case 'unapproved':
+				return createCommentNotice( 'is-info', translate( 'Comment unapproved.' ), options );
```

Comment:
Hi! I've found a possible matching string that has already been translated **97** times:
`translate( 'Unapproved' )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments/actions-transitions-notices)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).